cypress/e2e/universal-search.test.ts

```diff
@@ -1,13 +1,17 @@
 import { getTestSelector } from '../utils'
 
 describe('Universal search bar', () => {
-  before(() => {
-    cy.visit('/')
+  function openSearch() {
     cy.get('[data-cy="magnifying-icon"]')
```

Comment:
This should be updated. If we expect search to be accessible via `/`, I'd prefer it be accessed it that way. [Data attributes should generally be considered brittle and only used when necessary](https://github.com/Uniswap/interface/blob/main/cypress/README.md#prefer-selecting-elements-using-on-screen-text-over-data-testid-attributes).